docs/source/app_developers_guide/intro_xo_transaction_family.rst

```diff
@@ -1,228 +1,238 @@
 
-Playing XO with the XO Client (Example)
-=======================================
-
-Now that we have gone through the basics of the *XO* transaction processor, we are ready
-to play a game. The steps below show you how to set up and play a game using the *XO* CLI.
+     The following example specifies the URL for the Docker demo application
+     environment when creating a new game:
 
-Start the Necessary Components
-------------------------------
+     .. code-block:: console
 
-To play *XO*, ensure that the following components are running and connected:
+        $ xo create my-game --username jack --url http://rest-api:8008
 
-#. At least one validator
-#. An *XO* family transaction processor
-#. The REST API
 
-Create Players
---------------
+Step 1. Create Players
+----------------------
 
 Create keys for two players to play the game:
 
 .. code-block:: console
 
     $ sawtooth keygen jack
-    $ sawtooth keygen jill
-
-
-The command produces output similar to the following for both players:
-
-.. code-block:: console
-
     writing file: /home/ubuntu/.sawtooth/keys/jack.priv
     writing file: /home/ubuntu/.sawtooth/keys/jack.addr
+
+    $ sawtooth keygen jill
     writing file: /home/ubuntu/.sawtooth/keys/jill.priv
     writing file: /home/ubuntu/.sawtooth/keys/jill.addr
 
 
-Create a Game
--------------
+.. note::
 
-Create a game with the following command:
+   The output may differ slightly from this example.
 
-.. code-block:: console
 
-    $ xo create game --username jack
+Step 2. Create a Game
+---------------------
 
-To see list of the created games, enter the following command:
+Create a game named ``my-game`` with the following command:
 
 .. code-block:: console
 
-    $ xo list
+    $ xo create my-game --username jack
+
+.. note::
 
-The command outputs a list of the games that have been created:
+   The ``--username`` argument is required for ``xo create`` and ``xo take``
+   so that a single user (you) can play as two players. By default,
+   ``<username>`` is the Linux user name of the person playing the game.
+
+Verify that the ``create`` transaction was committed by displaying the list of
+existing games:
 
 .. code-block:: console
 
+    $ xo list
     GAME            PLAYER 1        PLAYER 2        BOARD     STATE
-    game                                            --------- P1-NEXT
+    my-game                                         --------- P1-NEXT
+
+.. note::
 
+   The ``xo list`` command is a wrapper that provides a quick way to show game
+   state rather than using ``curl`` with the REST API's URL to request state.
 
-Take a Space as Player One
---------------------------
 
-Start playing by taking a space as the first player, "jack":
+Step 3. Take a Space as Player 1
+--------------------------------
+
+The first player to issue an ``xo take`` command to a newly created game is
+recorded by username (Linux user name) as ``PLAYER 1``. The second player
```

Comment:
The default username for both `sawtooth keygen` and the `xo --username` option is the "login name": [https://docs.python.org/2/library/getpass.html#getpass.getuser](url). 

If you're running as root and do `sawtooth keygen` and `xo take my-game 5`, the name of your keyfile will be root and `xo` will look at `~/.sawtooth/keys/root.priv` to find your private key to sign the transaction with. However, if you do `sawtooth keygen jack` and you want to use that key to sign the transaction, you have to use the `--username jack` option. The players are recorded by their public key.

Reply:
@dplumb94 : Very helpful - thanks!  How about if I change the note to say this? 

     The ``--username`` argument is required for ``xo create`` and ``xo take``
     so that the ``xo`` client knows where to look for the player's key file.

I'd like handle the full details in a separate PR (and perhaps in a different section, like the CLI Command Reference), because this procedure is intended to have "just enough" information. Does that sound okay?